Dockerfile

```diff
@@ -9,6 +9,8 @@ RUN curl -L https://github.com/coreos/etcd/releases/download/v${ETCD_VERSION}/et
     rm -rf etcd.tar.gz etcd-*
 
 COPY run.sh /bin/
+COPY healthcheck.sh /bin/
+RUN chmod 755 /bin/healthcheck.sh
```

Comment:
this line is not needed if you already set the file permission before committing it (of afterwards, to fix it)